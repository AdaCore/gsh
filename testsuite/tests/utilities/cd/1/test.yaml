title: Test for cd utility - degraded cases - testing error messages
