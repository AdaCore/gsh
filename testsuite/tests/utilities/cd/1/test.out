unknow option: cd -q dir_a
cd: -q: No such file or directory

unknown dir: cd unknown_dir
cd: unknown_dir: No such file or directory

cd on file: cd a.txt
cd: a.txt: No such file or directory

'-' when no oldpwd: cd -
cd: OLDPWD not set
